tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_ocsp.yml

```diff
@@ -9,6 +9,13 @@
   notify:
     - STOPOCSP
 
+- name: Set debug log level to 5
+  lineinfile:
+    path: /var/lib/pki/{{variable.OCSP_INSTANCE_NAME}}/conf/ocsp/CS.cfg
```

Comment:
Minor: same comment as before

Reply:
Currently i have excluded the playbook changes for another MR. So marking this comment resolved for this MR.
This MR consists changes related to executing existing tests in Fedora32.